npm/vue2/src/index.ts

```diff
@@ -363,10 +363,12 @@ export const mount = (
   })
   .then((win) => {
     if (optionsOrProps.log !== false) {
-      Cypress.log({
-        name: 'mount',
-        message: [message],
-      }).snapshot('mounted').end()
+      Vue.nextTick(() => {
```

Comment:
I'd expect the `nextTick` to be wrapped within the chain so as to ensure the tick and log is captured before the next Cypress command is fired. You could convert this function to be async and `await Vue.nextTick()` as the end.

Reply:
I am not sure we need async/await - could just do it like this, right?

```js
.then((win) => Vue.nextTick())
.then(() => {
  /* stuff */
})
```
____

If anyone was curious, under-the-hood `nextTick` is basically just `Promise.resolve` (with various fallbacks). It actually uses `MutationObserver` as a fallback, interestingly enough.

Reply:
As long as the `.then(() => Vue.nextTick())` is done after the component creation I'm cool with it!

Reply:
This sounds good. I've made the necessary changes on the latest commit